### PR TITLE
Added copy node config button

### PR DESCRIPTION
**Pull request by deregs**
# Description

Relates to issue: **[226](https://github.com/ZeusLN/zeus/issues/226)**

Added copy node config button below delete node config. Brings user to a new Node Configuration page with all of the same properties, except for ' copy' added to the end of the Nickname. 

This pull request is categorized as a:

- [x] New feature
- [ ] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Other

## Checklist
- [x] I’ve run `npm run tsc` and made sure my code compiles correctly
- [x] I’ve run `npm run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `npm run prettier` and made sure my code is formatted correctly
- [x] I’ve run `npm run test` and made sure all of the tests pass

## Testing

If you added new functionality or fixed a bug, did you add new unit tests?

- [x] No, I’m a fool
- [ ] Yes
- [ ] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [ ] Android
- [x] iOS iphone 11 Pro iOS 13.7

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [x] LND
- [x] c-lightning-REST
- [x] spark
- [x] Eclair
- [ ] LndHub

### Locales
- [x] I’ve added new locale text that requires translations
- [x] I’m aware that new translations should be made on the Zeus [Transfix page](https://www.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `npm install` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `package-lock.json` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: views/Settings/AddEditNode.tsx [hunk 1]
```diff
@@ -1231,6 +1289,25 @@ export default class AddEditNode extends React.Component<
...
                     )}
+
+                    {saved && (
+                        <View style={styles.button}>
+                            <Button
+                                title={localeString(
+                                    'views.Settings.AddEditNode.copyNode'
+                                )}
+                                onPress={() => {
+                                    this.refs._scrollView.scrollTo({
```
Comment (kaloudis) [inline views/Settings/AddEditNode.tsx:1300]: maybe add a comment here

File: views/Settings/AddEditNode.tsx [hunk 2]
```diff
@@ -300,6 +300,63 @@ export default class AddEditNode extends React.Component<
...
+            accessKey,
+            username,
+            password,
+            implementation,
+            certVerification
+        } = this.state;
+        const { nodes, lurkerMode, passphrase, fiat, locale } = settings;
+
+        const node = {
+            nickname: nickname + ' copy',
```
Comment (kaloudis) [inline views/Settings/AddEditNode.tsx:324]: nit: ``${nickname} copy``

File: views/Settings/AddEditNode.tsx (hunk 1 above)
Comment (deregs) [reply views/Settings/AddEditNode.tsx:1305]: done, is the comment clear?

File: views/Settings/AddEditNode.tsx (hunk 2 above)
Comment (deregs) [reply views/Settings/AddEditNode.tsx]: done

**Comment by kaloudis** [comment]
This is great! @Bosch-0 curious on your take here. I think Copy should be above the delete button but will defer to Bosch

**Comment by Bosch-0** [comment]
Above would be better with the current UI.

Will be working on the settings frames this week, will come back and share something a bit cleaner - currently too many buttons on this page imo

**Comment by deregs** [comment]
Yeah agreed there are too many buttons on this page, but moved copy above delete for now.

**Review by kaloudis — APPROVED** [review]

**Pull request merged**